### PR TITLE
Mention SPIP in README

### PR DESCRIPTION
**Pull request by DarthMax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,18 @@ CAPS is under rapid development and we are planning to offer support for:
 - integration with Spark SQL
 - injection of custom graph data sources
 
+## Spark Improvement Proposal
+
+Currently CAPS is a third party add-on to the Spark eco system. We, however, believe that property graphs and graph processing
+has the potential to be come a vital part of data analytical processes. We are thus working, in cooperation with
```
Comment (s1ck) [inline README.md]:
```suggestion
has the potential to be come a vital part of data analytics. We are thus working, in cooperation with 
```

File: README.md [hunk 2]
```diff
@@ -49,6 +49,18 @@ CAPS is under rapid development and we are planning to offer support for:
 - integration with Spark SQL
 - injection of custom graph data sources
 
+## Spark Improvement Proposal
+
+Currently CAPS is a third party add-on to the Spark eco system. We, however, believe that property graphs and graph processing
+has the potential to be come a vital part of data analytical processes. We are thus working, in cooperation with 
+*Databricks*, on making CAPS a core part of Spark. 
+The first step on this road is the specification of a __PropertyGraph API__, similar to __SQL__ and __Dataframes__, along with porting
+the basic features of CAPS to the core Spark project in a so called __Spark Improvement Proposal__ (SPIP).
```
Comment (s1ck) [inline README.md]:
```suggestion
Cypher 9 features of CAPS to the core Spark project in a so called __Spark Improvement Proposal__ (SPIP).
```

File: README.md [hunk 3]
```diff
@@ -49,6 +49,18 @@ CAPS is under rapid development and we are planning to offer support for:
...
+We are currently in the first face of this process. The SPIP describing the motivation and goals is published here
```
Comment (s1ck) [inline README.md]:
```suggestion
We are currently in the first phase of this process. The SPIP describing the motivation and goals is published here
```

File: README.md (hunk 2 above)
Comment (Mats-SX) [inline README.md]:
```suggestion
the basic features of CAPS to the core Spark project in a so called __Spark Project Improvement Proposal__ (SPIP).
```

Comment (DarthMax) [reply README.md]:
```suggestion
Cypher 9 features of CAPS to the core Spark project in a so called __Spark Project Improvement Proposal__ (SPIP).
```

Comment (DarthMax) [reply README.md]: Applied through Martins suggestions

**Pull request merged**